mnefun/_mnefun.py

```diff
@@ -1419,7 +1419,21 @@ def save_epochs(p, subjects, in_names, in_numbers, analyses, out_names,
             evokeds = list()
             for name in names:
                 if p.average_weighting == 'chpi':
-                    weights = _chpi_snr_epochs(e[name])
+                    # read in raw unfiltered files for cHPI weighting evokeds
+                    raw_names = get_raw_fnames(p, subj, 'raw', False, True,
+                                               run_indices[si])
+                    raw = [Raw(fname, preload=False, allow_maxshield=True)
+                           for fname in raw_names]
+                    raw_unfil = concatenate_raws(raw)
```

Comment:
@Eric89GXL `_raw_LRFCP` didn't really behave here in terms of filtering and resampling. Wondering if similar reasons prevented usage in ln 1331 of `save_epochs`?